donkeycar/templates/calibrate.py

```diff
@@ -113,22 +115,14 @@ def drive(cfg ):
         drive_train = RoboHATDriver(cfg)
         V.add(drive_train, inputs=['angle', 'throttle'])
 
-
+    # TODO: monkeypatching is bad!!!
```

Comment:
I think those values are not even used anywhere in the controller or here in the calibration template.  Maybe some other code uses them; you right - this is very hacky.